src/components/alert/README.md

```diff
@@ -44,17 +44,19 @@ Alerts are available for any length of text, as well as an optional dismiss butt
 
 <script>
 export default {
-  data: {
-    dismissSecs: 10,
-    dismissCountDown: 0,
-    showDismissibleAlert: false
+  data () {
```

Comment:
Technically, the examples are run as app instance (each example is a `new Vue(...)`), rather than component instances, so `data` as a function is not required.